presto-main/src/main/java/com/facebook/presto/type/DoubleOperators.java

```diff
@@ -175,7 +175,7 @@ public static long castToInteger(@SqlType(StandardTypes.DOUBLE) double value)
             return Ints.checkedCast((long) MathFunctions.round(value));
         }
         catch (IllegalArgumentException e) {
-            throw new PrestoException(NUMERIC_VALUE_OUT_OF_RANGE, e);
+            throw new PrestoException(NUMERIC_VALUE_OUT_OF_RANGE, "Out of range for integer: " + value, e);
```

Comment:
Shouldn't we be more specific here, e.g. "_Out of range for integer when casting from double_"? For larger queries it may not be very obvious where out-of-range occurred.